File: src/test/java/com/werfad/KeyTagsGeneratorTest.java
package com.werfad;

import org.junit.Test;

import java.util.List;

import static org.junit.Assert.assertEquals;

public class KeyTagsGeneratorTest {

    @Test
    public void createTagsTree() {
        List<String> res = KeyTagsGenerator.createTagsTree(756,
                "abcdefghijklmnopqrstuvwxyz");

        assertEquals(res.size(), 756);
        System.out.println(res.size());
        System.out.println(res);

        List<String> res1 = KeyTagsGenerator.createTagsTree(0,
                "abcdefghijklmnopqrstuvwxyz");
        assertEquals(res1.size(), 0);
    }
}

File: test/com/werfad/utils/KeyTagsGeneratorTest.kt
package com.werfad.utils

import org.junit.Assert.assertEquals
import org.junit.Test

class KeyTagsGeneratorTest {

    @Test
    fun testCreateKeyTagsTree() {
        val res = KeyTagsGenerator.createTagsTree(756, "abcdefghijklmnopqrstuvwxyz")

        assertEquals(res.size.toLong(), 756)
        println(res.size)
        println(res)
    }
}